Assets/Scripts/UI/Pickup Pup/DogBrowser.cs

```diff
@@ -0,0 +1,332 @@
+﻿/*
+ * Author(s): Isaiah Mann
+ * Description: Controls the display of a group of dogs in an organized UI
+ * Usage: Should be attached to the parent element in the UI that holds the child elements
+ */
+
+using UnityEngine;
+
+public class DogBrowser : PPUIElement 
```

Comment:
Do you plan on revising this script at all? It's a bit long. Can come later though.